src/plugins/intel_gpu/src/kernel_selector/kernels/sdpa/sdpa_kernel_micro.cpp

```diff
@@ -504,11 +504,12 @@ JitConstants SDPAKernelMicro::GetJitConstants(const sdpa_params& params, const m
     const auto& K = prim_params.inputs[1];
     const auto& V = prim_params.inputs[2];
 
-    const auto head_size = prim_params.conf.head_size;
+    const auto head_size = prim_params.conf.k_head_size;
+    const auto v_head_size = prim_params.conf.k_head_size;
```

Comment:
Please check if it's right.
```suggestion
    const auto v_head_size = prim_params.conf.v_head_size;
```

Reply:
You are right.  Updated.